thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/DetectionJobRunner.java

```diff
@@ -120,7 +110,7 @@ public void execute(JobExecutionContext jobExecutionContext) throws JobExecution
   private DateTime alignTimestampsToDataTimezone(DateTime inputDateTime, String collection) {
 
     try {
-      DatasetConfigDTO datasetConfig = datasetConfigDAO.findByDataset(collection);
+      DatasetConfigDTO datasetConfig = DB.getDatasetConfigDAO().findByDataset(collection);
```

Comment:
Please make the variable names consistent. at one place it's DB, another its DAO_REGISTRY.